tools/goctl/api/swagger/contenttype.go

```diff
@@ -11,14 +11,16 @@ func consumesFromTypeOrDef(ctx Context, method string, tp spec.Type) []string {
 	if strings.EqualFold(method, http.MethodGet) {
 		return []string{}
 	}
+
```

Comment:
`consumesFromTypeOrDef` this function need not changed.

Reply:
understand, I have corrected it